tests/latest/valid/dbt_yml_files.yml

```diff
@@ -0,0 +1,178 @@
+# this file was generated with dbt init with dbt 1.2.1
+
+models:
+  - name: my_first_dbt_model
+    description: "A starter dbt model"
+    config:
+      grants:
+        privilege: role
+        privileges: ["select_user", "select_cruiser"]
+    columns:
+      - name: id
+        description: "The primary key for this table"
+        tests:
+          - unique
+          - not_null
+
+  - name: my_second_dbt_model
+    description: "A starter dbt model"
+    docs:
+      show: true
+      node_color: yellow
+    columns:
+      - name: id
+        description: "The primary key for this table"
+        tests:
+          - unique
+          - not_null
+
+  - name: my_contracted_dbt_model
+    description: "A dbt model with contracts"
+    access: public
+    group: analytics
+    latest_version: 2
+    config:
+      contract: 
+        enforced: true
+    columns:
+      - name: id
+        description: "The primary key for this table"
+        data_type: int
+        constraints: 
+          - type: not_null
+          - type: unique
+          - type: check
+            expression: (id > 0)
+        tests:
+          - unique
+          - not_null
+    versions:
+      - v: 2
+        columns:
+          - include: '*'
+            exclude: country_name
+          - name: id  # included in addition the '*' set. if id were in the '*' set -> override it
+            description: This is the primary key
+            data_type: float
+      - v: 1
+        config:
+          alias: dim_customers
+
+  - name: my_model_level_contract_model
+    config:
+      contract:
+        enforced: true
+    constraints:
+    - type: check
+      expression: (id > 0)
+    - type: primary_key
+      columns: [ id ]
+    - type: unique
+      columns: [ color, date_day ]
+      name: strange_uniqueness_requirement
+  
+  - name: my_materialized_view
+    config:
+      materialized: materialized_view
+      auto_refresh: true
+      backup: '{{ target.name == "prod" }}'
+      target_lag: downstream
+      snowflake_warehouse: my_warehouse
+
+unit_tests:
+  - name: my_first_unit_test
+    expect:
+      fixture: a_seed_that_points_to_expected_output
+      format: csv
+    given:
+      - input: ref('model_b')
+        rows:
+          - {id: 1, code: 'abc'}
+          - {id: 2, code: 'def'}
+    model: my_first_dbt_model
+  
+  - name: my_second_unit_test
```

Comment:
can we add an inline csv to any of the unit tests?

Reply:
yeh good idea